src/coap_session.c

```diff
@@ -27,50 +27,100 @@
 #include <errno.h>
 
 void
-coap_session_set_max_retransmit (coap_session_t *session, unsigned int value) {
-  if (value > 0)
-    session->max_retransmit = value;
-  coap_log(LOG_DEBUG, "***%s: session max_retransmit set to %d\n",
-           coap_session_str(session), session->max_retransmit);
-  return;
-}
-
-void
-coap_session_set_ack_timeout (coap_session_t *session, coap_fixed_point_t value) {
-  if (value.integer_part > 0 && value.fractional_part < 1000)
+coap_session_set_ack_timeout(coap_session_t *session, coap_fixed_point_t value) {
+  if (value.integer_part > 0 && value.fractional_part < 1000) {
     session->ack_timeout = value;
-  coap_log(LOG_DEBUG, "***%s: session ack_timeout set to %d.%03d\n",
+    coap_log(LOG_DEBUG, "***%s: session ack_timeout set to %u.%03u\n",
            coap_session_str(session), session->ack_timeout.integer_part,
            session->ack_timeout.fractional_part);
+  }
   return;
```

Comment:
This return statement is not needed and may be cleaned up as part of this PR.

Reply:
Cleaned up in similar places.